src/dynamics/ads_player/ads_player.js

```diff
@@ -343,6 +347,22 @@ Scoped.define("module:Ads.Dynamics.Player", [
                         this._videoElement = this.parent() && this.parent().activeElement().querySelector("[data-video='video']"); // TODO video element for outstream
                     return this._videoElement;
                 },
+                renderVideoFrame: function(mediaUrl, width, height) {
+                    const video = document.createElement("video");
+                    const canvas = document.createElement("canvas");
+                    video.crossOrigin = "anonymous";
+                    video.src = mediaUrl;
+                    video.currentTime = this.get('currenttime');
+                    setTimeout(function() {
+                        canvas.width = width;
+                        canvas.height = height;
+                        canvas.getContext("2d")
+                            .drawImage(video, 0, 0, canvas.width, canvas.height);
+                        this.getAdContainer().style.backgroundImage = `url(${canvas.toDataURL("image/jpeg")})`;
+                        this.getAdContainer().style.backgroundRepeat = "no-repeat";
+                        this.getAdContainer().style.backgroundSize = "contain";
```

Comment:
Hi @maxblessed, I think these styles might get overridden by the `ba-styles` partial if it's value is changed:
https://github.com/betajs/betajs-media-components/blob/34819671286be7c383b518f39e55afd1c60fd938/src/dynamics/ads_player/ads_player.html#L5